### PR TITLE
diskDetailsTest

### PR DESCRIPTION
**Pull request by evgeniiamokhova**
added disk details without tests of creation template and reverting disk to snapshot

### CODE REVIEW

**Comment by evgeniiamokhova** [comment]
run tests

File: e2e/pages/disk-list.po.ts [hunk 1]
```diff
@@ -40,4 +40,21 @@ export class DiskList extends CloudstackUiPage {
   clickClose() {
     element(by.css('.backdrop.ng-star-inserted')).click();
   }
+
+  clickOnReadyDisk() {
+    element
+      .all(by.css('.entity-card.mat-card'))
+      .first()
```
Comment (rennervo) [inline e2e/pages/disk-list.po.ts]: In this method, you click on the first available disk, but you must click on the disk in the status "Ready"

File: e2e/pages/disk-sidebar.po.ts [hunk 2]
```diff
@@ -16,4 +21,88 @@ export class DiskSidebar extends CloudstackUiPage {
...
+      5000,
+      "Text area of disk description doesn't present",
+    );
+    element(by.name('textField')).clear();
+    element(by.name('textField')).sendKeys(description);
+    element
+      .all(by.css('.buttons button'))
+      .last()
+      .click();
+    element.all(by.css('.buttons button')).isPresent();
```
Comment (rennervo) [inline e2e/pages/disk-sidebar.po.ts]: Why do you need the last line of code in this function? Maybe it is superfluous?

Comment (evgeniiamokhova) [reply e2e/pages/disk-sidebar.po.ts]: Here I check the button "Cancel", but I incorrectly pointed to it

**Pull request merged**